config/crd/bases/dscinitialization.opendatahub.io_dscinitializations.yaml

```diff
@@ -81,6 +81,121 @@ spec:
                     description: Namespace for monitoring if it is enabled
                     type: string
                 type: object
+              serverless:
+                description: Configures Serverless (KNative Serving). This is a prerequisite
+                  for single model serving (KServe) and you should review this configuration
+                  if you are planning to use KServe.
+                properties:
+                  managementState:
+                    default: Removed
+                    enum:
+                    - Managed
+                    - Removed
+                    pattern: ^(Managed|Unmanaged|Force|Removed)$
```

Comment:
@VaishnaviHire @zdtsw it applies to most of the things here so it's not a comment for this particular PR... the way we use states now makes enum and pattern not matching each other. I wonder which takes precedence here and if we shouldn't align it.

Reply:
the old plan was to not have enum in the end when we implemented Unmanaged (not sure if we will need Force or not in long term) but since we do not want user be able to put Unmanaged or Force for the time being, we added enum to force with 2 options.
My personal opinion: once logic for Unmanaged is done, we can remove enum